doc/classes/Basis.xml

```diff
@@ -242,8 +248,8 @@
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Returns this basis rotated around the given [param axis] by [param angle] (in radians). The [param axis] must be a normalized vector (see [method Vector3.normalized]).
-				Positive values rotate this basis clockwise around the axis, while negative values rotate it counterclockwise.
+				Returns a copy of this basis rotated around the given [param axis] by the given [param angle] (in radians).
+				The [param axis] must be a normalized vector (see [method Vector3.normalized]). If [param angle] is positive, the basis is rotated clockwise around the axis.
```

Comment:
While I did solve this, I'm keeping it open if you consider the additional notes here to be noteworthy enough that they should be mentioned in the `rotated()` methods somehow.

Reply:
I'm not sure if they should, it's not really something that helps the user understand, I don't want to overwhelm the user with details. It's easier to tell the user "it works this way" instead of "the math is the same either way but in Godot it works this way". I'll leave it up to you.